test/PooltogetherGovernor.t.sol

```diff
@@ -295,4 +296,465 @@ contract Propose is ProposalTest {
     assertEq(_token.balanceOf(TIMELOCK), _timelockTokenBalance);
     assertEq(_token.balanceOf(_receiver), _receiverTokenBalance);
   }
+
+  ////
+  // Bravo proposal tests
+  ////
+  function testFuzz_NewGovernorCanReceiveNewProposal(uint256 _poolAmount, address _poolReceiver)
+    public
+  {
+    _assumeReceiver(_poolReceiver);
+    _upgradeToBravoGovernor();
+    _submitTokenSendProposalToGovernorBravo(POOL_TOKEN, _poolAmount, _poolReceiver);
+  }
+
+  function testFuzz_NewGovernorCanDefeatProposal(uint256 _amount, address _receiver, uint256 _seed)
+    public
+  {
+    IERC20 _token = _randomERC20Token(_seed);
+    _assumeReceiver(_receiver);
+
+    _upgradeToBravoGovernor();
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildTokenSendProposal(address(_token), _amount, _receiver);
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets,
+      _values,
+      _calldatas,
+      _description,
+      (_amount % 2 == 1 ? AGAINST : ABSTAIN) // Randomize vote type.
+    );
+
+    // It should not be possible to queue the proposal
+    vm.expectRevert("Governor: proposal not successful");
+    governorBravo.queue(_targets, _values, _calldatas, keccak256(bytes(_description)));
+  }
+
+  function testFuzz_NewGovernorCanPassProposalToSendToken(
+    uint256 _amount,
+    address _receiver,
+    uint256 _seed
+  ) public {
+    IERC20 _token = _randomERC20Token(_seed);
+    _assumeReceiver(_receiver);
+    uint256 _timelockTokenBalance = _token.balanceOf(TIMELOCK);
+
+    // bound by the number of tokens the timelock currently controls
+    _amount = bound(_amount, 0, _timelockTokenBalance);
+    uint256 _initialTokenBalance = _token.balanceOf(_receiver);
+
+    _upgradeToBravoGovernor();
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildTokenSendProposal(address(_token), _amount, _receiver);
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+
+    // Ensure the tokens have been transferred
+    assertEq(_token.balanceOf(_receiver), _initialTokenBalance + _amount);
+    assertEq(_token.balanceOf(TIMELOCK), _timelockTokenBalance - _amount);
+  }
+
+  function testFuzz_NewGovernorCanPassProposalToSendETH(uint256 _amount, address _receiver) public {
+    _assumeReceiver(_receiver);
+    vm.deal(TIMELOCK, _amount);
```

Comment:
Since one of the things PoolTogether is explicitly worried about, let's add a test that is basically a clone of this one, but does *not* `deal` to the `TIMELOCK`. This will explicitly test that the currently locked ETH held by the Timelock can be transferred. Perhaps overkill, but easy to do so why not.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/a44bd61cfde59a23f71cf0051f37ce4789a0a208